data/outfits/god.txt

```diff
@@ -1,36 +1,71 @@
 outfit "Ruin-The-Fun God Mode"
 	"display name" "RTF God Mode"
 	category `RTF God Outfits`
-	cost 0
 	thumbnail "outfit/quarg quantum shield generator"
-	"mass" 0
 	hull 1000000
 	"hull repair rate" 1000000
-	"hull energy" 0
 	shields 1000000
 	"shield generation" 1000000
-	"cooling" 1000000
-	"ion resistance" 1000000
+	"high shield permeability" 0
+	"low shield permeability" 0
 	"disruption resistance" 1000000
+	"ion resistance" 1000000
+	"scramble resistance" 1000000
+	"slowing resistance" 1000000
 	"discharge resistance" 1000000
+	"corrosion resistance" 1000000
+	"leak resistance" 1000000
+	"burn resistance" 1000000
+	"piercing resistance" 1000000
+	"disruption protection" 1000000
+	"energy protection" 1000000
+	"force protection" 1000000
+	"fuel protection" 1000000
+	"heat protection" 1000000
+	"hull protection" 1000000
+	"ion protection" 1000000
+	"scramble protection" 1000000
 	"piercing protection" 1000000
+	"shield protection" 1000000
 	"slowing protection" 1000000
 	"discharge protection" 1000000
 	"corrosion protection" 1000000
 	"leak protection" 1000000
 	"burn protection" 1000000
-	bunks 1000000
-	description "Absolutely illegal."
+	description "Protect your ship against almost anything."
+
+outfit "Ruin-The-Fun God Scanner"
+	"display name" "RTF God Scanner"
+	category "RTF God Outfits"
+	thumbnail "outfit/outfit scanner"
+	"tactical scan power" 100000000
+	"asteroid scan power" 100000000
+	"outfit scan power" 100000000
+	"cargo scan power"  100000000
+	"outfit scan efficiency" 1000000
+	"cargo scan efficiency" 1000000
+	description "Scans far and wide."
+
+outfit "Ruin-The-Fun God Engine"
+	"display name" "RTF God Engine"
+	category "RTF God Outfits"
+	thumbnail "outfit/tiny atomic thruster"
+	"thrust" 200
+	"reverse thrust" 200
+	description "Runs really fast."
 
+outfit "Ruin-The-Fun God Steering"
+	"display name" "RTF God Steering"
+	category "RTF God Outfits"
+	thumbnail "outfit/tiny atomic steering"
+	"turn" 5000
+	description "Turns really fast."
```

Comment:
To better justify those outfits being God outfits rather than single stats, it would be nice giving them a flare effect like vanilla engines and thruster have.

I would give them one of the existing vanilla engine/steering effects. I would not add new images, to not increase the plugin size.